modules/@apostrophecms/task/index.js

```diff
@@ -67,7 +64,7 @@ module.exports = {
           }
 
           try {
-            await task.fn(self.apos, self.apos.argv);
+            await task.task(self.apos.argv);
```

Comment:
It seems odd written here, but when the tasks are added directly as keys of a `tasks` object in a module with keys `usage` and `task` it makes more sense. It's better to have keys with full names rather than an abbreviation like `fn`.

```
tasks: {
 myTask: {
    usage: 'do things',
    task: async function (argv) { ... }
  }
} 
```

versus 
```
tasks: {
 myTask: {
    usage: 'do things',
    fn: async function (argv) { ... }
  }
} 
```